pulp_container/app/serializers.py

```diff
@@ -134,11 +134,11 @@ class ContainerRemoteSerializer(RemoteSerializer):
         allow_blank=False,
         help_text=_("Name of the upstream repository")
     )
-    whitelist_tags = serializers.ListField(
+    allowlist_tags = serializers.ListField(
         child=serializers.CharField(max_length=255),
         allow_null=True,
         required=False,
-        help_text=_("A list of whitelisted tags to sync")
+        help_text=_("A list of allowlisted tags to sync")
```

Comment:
maybe `allowed tags`?

Reply:
i would also suggest to consider `allowed_tag_list`